src/js/libpannellum.js

```diff
@@ -101,6 +101,37 @@ function Renderer(container) {
         pose = undefined;
 
         var s;
+        var faceMissing = false;
+        var cubeImgWidth = undefined;
```

Comment:
The `= undefined` part is unnecessary; `var cubeImgWidth;` is functionally identical.